.circleci/config.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             circleci-agent step halt
           fi
 
-      - run: golang/scripts/build.sh
+      - run: golang/scripts/build.sh kurtosistech/kurtosis-lambda-starter-pack
```

Comment:
oo I don't think `build.sh` should take in a parameter, since users would need to specify that every time they run. IMO `build.sh` should have the name of the image to be built hardcoded inside